.github/workflows/checks.yml

```diff
@@ -24,3 +24,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
       - uses: ./.github/actions/security/lockfile
+        with:
+          allowedHosts:
```

Comment:
Just a question, am assuming this is to allow ts-zen fork to pass the security check ?

Reply:
Exactly
btw ping @alexandrebodin